### ISSUE TITLE
.vscode settings.json hides the awscloudformation folder

### ISSUE THREAD

**Issue by ykethan** [aws-amplify/amplify-cli#13298]
### Amplify CLI Version

12.5.1

### What operating system are you using?

Mac

### Did you make any manual changes to the cloud resources managed by Amplify? Please describe the changes made.

no

### Describe the bug

On a `amplify init` the `.vscode` folder is created which hides couple of folders and files including the `amplify/backend/awscloudformation`. 

When running `amplify override project` the `override.ts` file get created under the `awscloudformation` folder. Which becomes hidden if the user is not aware of `.vscode` folder settings

### Expected behavior

remove the flag if project override is used.

### Reproduction steps

1. amplify init 
2. amplify override project
3. in vscode observe the override file under `awscloudformation` folder

### Log output

<details>

```
# Put your logs below this line


```

</details>

### Before submitting, please confirm:

- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.
- [X] I have removed any sensitive information from my code snippets and submission.

**Comment by josefaidt** [comment]
Confirmed this is a bug. CLI generates the following settings file, which effectively hides this directory from the developer

```json
{
  "files.exclude": {
    "amplify/.config": true,
    "amplify/**/*-parameters.json": true,
    "amplify/**/amplify.state": true,
    "amplify/**/transform.conf.json": true,
    "amplify/#current-cloud-backend": true,
    "amplify/backend/amplify-meta.json": true,
    "amplify/backend/awscloudformation": true
  }
}
```

**Comment by rtpascual** [comment]
Fix is done in #13310
